### ISSUE TITLE
None for SIDS after RWY 21 selected at GMME 

### ISSUE THREAD

**Issue by Pat15250** [flybywiresim/a32nx#5369]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-07-21T23:15:28+00:00",
  "ref": "master",
  "sha": "ce44f46607ba7f309c53d4be0b5f371dee7948b3",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

Hello, 
This morning, I planned a flight from GMME to GCXO.
On the FMC, after init flight (GMME - GCXO - cost index (60) - and flight level (FL330), I selected RWY 21 for the departure, and SIDS was missed.
Congratulations for your work.
Best regards.
Patrick

### Expected behavior

I expected to have SID list after having selected RWY21 at GMME.

### Steps to reproduce

GMME
init completed on FMC
RWY 21 selected at GMME

**Comment by donstim** [comment]
Like this?
![Screenshot (1026)](https://user-images.githubusercontent.com/70166617/126851246-db37962d-c92c-47d8-9716-730889897fb8.png)

No issue for me. However, I use Navigraph for nav data. Do you have similar issues at other airports, or only GMME?

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
